ccf_transaction_processor/scripts/start_ccf_network.sh

```diff
@@ -44,6 +44,11 @@ try ${F_SERVICEHOME}/bin/start_cchost.sh
 
 sleep 5
 
+if [ $SGX_MODE == 'HW' ]; then
+    say get quote, retrieve mrenclave and compare against expected mrenclave
+    try ${F_SERVICEHOME}/bin/get_quote_and_verify_mrenclave.py --logfile __screen__ --oesign ${CCF_BASE}/openenclave/bin/oesign
+fi
+
```

Comment:
Wouldn't this code rather have to run on the PDO side/client side rather than on the TP side as now?  On the TP-side this validation is not really useful (from a security perspective) as you are validating yourself?